scripts/provision.sh

```diff
@@ -65,15 +85,18 @@ configure() {
 
   # don't write out the builder-minio user.toml if using S3 or Artifactory directly
   if [ "${S3_ENABLED:-false}" = "false" ] && [ "${ARTIFACTORY_ENABLED:-false}" = "false" ]; then
```

Comment:
It's not part of this pr, but should these also be `==`?

Reply:
Nope. As I understand it, the `[]` in bash are a synonym for `test` and for the `test` syntax this is correct.